src/Repository/QuizTemplateRepository.php

```diff
@@ -46,11 +46,11 @@ public function getSelectedQuestions($quizid)
         return $result;
     }
 
-    public function deleteQuestions($quizid)
+    public function deleteQuestions($quizId)
```

Comment:
return type (my guess is you need to  add a lot of return types, so please take a look at all classes)